src/script/components/manifest-options.ts

```diff
@@ -895,11 +913,35 @@ export class AppManifest extends LitElement {
     }
   }
 
+  /**
+   * Syncs the form state with the preview component.
+   * 
+   * @param fieldName - The input name that's currently being modified
+   */
+  handlePreviewerSync(fieldName: string) {
+    switch (fieldName) {
+      case 'name': 
+        this.previewStage = 'name';
+        break;
+      case 'short_name':
+        this.previewStage = 'shortName';
+        break;
+      case 'display':
+        this.previewStage = 'display';
+        break;
+      case 'theme_color':
+        this.previewStage = 'themeColor';
```

Comment:
I think we changed these back to the snake case here, is that not what you are seeing?